.github/workflows/ci-pre-commit.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: pre-commit hooks
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3.3.0
+      - uses: actions/checkout@v3
```

Comment:
Our current setup is to specify the full version and use dependabot to automatically open PRs to update versions when a new one comes out. I think this will go against that approach. Is there a reason to prefer this instead?

Reply:
Ah, whoops, I missed @fjetter's similar comment here https://github.com/dask/distributed/pull/7470#discussion_r1068102582